ingestion/src/metadata/ingestion/source/dashboard/mode/client.py

```diff
@@ -180,11 +175,3 @@ def get_all_data_sources(self, workspace_name: str) -> Optional[dict]:
             logger.warning(f"Error fetching all data sources: {exc}")
 
         return None
-
-    def get_user_account(self) -> dict:
```

Comment:
The method was only being used for `test connection` and not in actual ingestion. Changed the test connection to actually test getting the dashboards now.
Also this `/account` endpoint is not supported for the `workspace tokens` as pointed out in the mode [docs](https://mode.com/developer/api-reference/management/users/#getCurrentUser)